src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -2011,7 +2012,9 @@ private BoundFunctionPointerInvocation BindFunctionPointerInvocation(SyntaxNode
             CoerceArguments(
                 methodResult,
                 analyzedArguments.Arguments,
-                diagnostics);
+                diagnostics,
+                boundExpression.Type,
```

Comment:
>boundExpression [](http://example.com/codeflow?start=16&length=15)

Is this the pointer being invoked? I am not sure if we should treet it as a receiver.